tools/travis-common-vars.sh

```diff
@@ -45,3 +48,26 @@ function mktempdir
 {
     mktemp -d "/tmp/$1.XXXXXXXXX"
 }
+
+function init_docker
+{
+    if [ "$REMOTE_RO_VOLUMES" = 'true' ]; then
+        export REMOTE_ROOT_DIR="$(mktempdir remote_mongooseim)"
+        docker rm -f mongooseim-uploader || echo "Skip mongooseim-uploader removal"
+        echo "Start remote uploader helper"
+        docker run --name mongooseim-uploader -v "$REMOTE_ROOT_DIR":/data -it busybox true
```

Comment:
Why exactly do we need this `mongooseim-uploader` container?

Reply:
We can't copy onto volume, that no container uses.
To copy we need to create a container with volume connected. Than we can copy onto it even if the container is not running.

We mount `$REMOTE_ROOT_DIR` twice, once in uploader container and once in the main service container.

Reply:
So, can you copy to $REMOTE_ROOT_DIR (that's on local file system, right?) and use it only once in the main service container. I still don't understand why we need to copy via the additional container.

Reply:
REMOTE_ROOT_DIR is on remote system. This code is to support docker client on one machine and docker server on another.

Docker will just create an empty directory in case it runs on docker-machine. Again, we don't really need the code, so we can remove support for DEFAULT_REMOTE_RO_VOLUMES from this PR.

Reply:
REMOTE_RO_VOLUMES removed.
Can be resurrected back once we need docker-machine.

Reply:
ok, cool. Thanks. I just tried this branch to set up MSSQL on my MacOSX and it worked smoothly! Waiting for travis to pass and it can be merged.